index.html

```diff
@@ -40,6 +41,170 @@ <h2>I'am a Software Developer</h2>
                 </div>
             </article>
         </section>
+        <section class="projects">
+            <header class="project-header">
+                <h2 class="projects-section-title">My Recent Works</h2>
+                <hr>
+            </header>
+            <article class="project-card">
```

Comment:
- [Optional] please make this a `<section>` or a `<div>` tag, so as to make your HTML more semantically correct